libs/ngtcp2/Makefile

```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ngtcp2
-PKG_VERSION:=1.1.0
-PKG_RELEASE:=1
+PKG_VERSION:=1.4.0
+PKG_RELEASE:=r1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/ngtcp2/ngtcp2/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=987d784643edea4f2859c405f7dfbc53871a9f7ae5fcddf5fb12ec5dfce1ef22
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://github.com/ngtcp2/$(PKG_NAME)/releases/download/v$(PKG_VERSION)/
```

Comment:
```suggestion
PKG_SOURCE_URL:=https://github.com/ngtcp2/ngtcp2/releases/download/v$(PKG_VERSION)/
```

Reply:
NACK: https://github.com/search?q=repo%3Aopenwrt%2Fpackages%20%24(PKG_NAME)%2Freleases&type=code

Reply:
While looking at it, I am not sure what you would like to say.  There are just a few packages looking at the URL, which you posted. Others are not related for PKG_SOURCE_URL, but nice attempt, I need to say! :) This conversation is not resolved and also, why you are merging it when I didnt have any chance to respond?

Reference:
https://github.com/openwrt/packages/pull/4119

I am sorry to ask, but what was your reason to change it? Make it harder to copy paste it to browser? I think so, because: https://github.com/ngtcp2/$(PKG_NAME)/ says 404. Your change makes it harder to read, to copy paste, well, for everything. There was no need to do it.

Reply:
So yes, avoid using `$(PKG_NAME)` for repo URL.

Reply:
@BKPepe @1715173329 do you gentlemen want to clean up all the other Makefiles containing `$(PKG_NAME)/releases` 
 in the `PKG_SOURCE_URL`?: https://github.com/search?q=repo%3Aopenwrt%2Fpackages%20%24(PKG_NAME)%2Freleases&type=code

Reply:
They were introduced for history reasons but let's not add new.